app/assets/javascripts/select-kit/addon/components/icon-picker.js

```diff
@@ -18,22 +23,36 @@ export default MultiSelectComponent.extend({
     this._super(...arguments);
 
     this._cachedIconsList = null;
+    this._resultCount = 0;
 
     if (isDevelopment()) {
       disableMissingIconWarning();
     }
+
+    this.insertAfterCollection(MAIN_COLLECTION, MORE_ICONS_COLLECTION);
+  },
+
+  modifyComponentForCollection(collection) {
+    if (collection === MORE_ICONS_COLLECTION) {
+      return "filter-for-more";
```

Comment:
Is it possible to import FilterForMore and use the reference here? Or does it break somewhere on the say to the `{{component}}` helper? 😃

Reply:
Yes, this works too.